src/domains/transaction/components/InputFee/InputFee.blocks.tsx

```diff
@@ -28,16 +28,16 @@ export const InputFeeSimple = ({
 				setSelectedValue={() => onChange(optionValue as InputFeeSimpleValue)}
 			>
 				<div>
```

Comment:
```suggestion
				<div className='flex flex-col space-y-2'>
```
why not? :hmhm:

Reply:
@w3ea thanks, didn't know of `space-*` classes. I added a commit with this change.